en.json

```diff
@@ -651,6 +688,31 @@
                     "title": "Change identity on {{server}}",
                     "nickname": "Nickname",
                     "avatar": "Avatar"
+                },
+                "voice": {
+                    "leave_call": "Leave Call",
+                    "mute_self": "Mute microphone",
+                    "unmute_self": "Unmute microphone",
+                    "undeafen": "Undeafen",
+                    "deafen": "Deafen"
+                },
+                "footer": {
+                    "discover_revolt": "Discover Revolt",
+                    "add_server": "Add a server"
+                },
+                "message": {
+                    "reply": "Reply",
+                    "react": "React",
+                    "edit": "Edit",
+                    "delete": "Delete",
+                    "more": "More",
+                    "mark_unread": "Mark as unread",
+                    "copied": "Copied!",
+                    "copy_link": "Copy link",
+                    "copy_id": "Copy ID"
```

Comment:
This does create some duplicates so  I would for now put these into the context menu actions object or  wherever it is and later merge all of these into a top-level `actions` key.

Reply:
Yes, I took a look at the duplicates but though that maybe it would be better to have different context for each of both (menu & tooltips)